### PR TITLE
More geographic subject mapping.

### LINKED ISSUE #1491: From fedora: Map geographic subject
For example druid:kt455nh2818:
```
<subject authority="lcsh" authorityURI="http://id.loc.gov/authorities/subjects/" valueURI="http://id.loc.gov/authorities/subjects/sh85005490">
        <geographic>Antarctica</geographic>
      </subject>
```

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1491

## Why was this change made?
More geographic subject mapping

## How was this change tested?
Unit, sdr-deploy

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by justinlittman** [comment]
Before:
```
Status (n=100000):
  Success:   80604 (80.604%)
  Different: 18724 (18.724%)
  To Cocina error:     33 (0.033%)
  To Fedora error:     29 (0.029%)
  Missing:     610 (0.61%)
```

After:
```
Status (n=100000):
  Success:   81972 (81.972%)
  Different: 17356 (17.356%)
  To Cocina error:     33 (0.033%)
  To Fedora error:     29 (0.029%)
  Missing:     610 (0.61%)
```

File: app/services/cocina/to_fedora/descriptive/subject.rb [hunk 1]
```diff
@@ -139,43 +139,42 @@ def write_classification(value, attrs)
           xml.classification value, attrs
         end
 
-        def write_topic(subject)
+        def write_topic(subject, is_parallel: false)
           case subject.type
           when 'person'
-            xml.name topic_attributes_for(subject).merge(type: 'personal') do
+            xml.name topic_attributes_for(subject, is_parallel: is_parallel).merge(type: 'personal') do
```
Comment (mjgiarlo) [inline app/services/cocina/to_fedora/descriptive/subject.rb:145]: This line looks uncovered. Annoying to add a test? Useful?

**Review by mjgiarlo — APPROVED** [review]
A question about code coverage but not one I'd blocked over. Approving.

File: app/services/cocina/to_fedora/descriptive/subject.rb (hunk 1 above)
Comment (justinlittman) [reply app/services/cocina/to_fedora/descriptive/subject.rb:145]: I looked for an example of this but couldn't find one. Let me see if one comes up in my testing.

**Pull request merged**